src/components/QuranReader/SidebarNavigation/SidebarNavigation.module.scss

```diff
@@ -28,7 +28,7 @@
   padding-block-start: calc(1.5 * var(--spacing-mega));
   background-color: var(--color-background-elevated);
   overflow-x: hidden;
-  z-index: var(--z-index-default);
+  z-index: var(--z-index-sticky);
```

Comment:
This is breaking current design for all non-Tajweed Musahfs

|Before|After| 
|---|---|
|<img width="1710" alt="Screenshot 2024-06-24 at 6 43 01 AM" src="https://github.com/quran/quran.com-frontend-next/assets/15169499/271ef046-670e-441c-bd4d-bec874d620fe">|<img width="1725" alt="Screenshot 2024-06-24 at 6 42 11 AM" src="https://github.com/quran/quran.com-frontend-next/assets/15169499/4bfe3487-189e-4fea-8c43-f249413fb99c">|